### PR TITLE
Add travel support

### PR DESCRIPTION
**Pull request by BenJam**
Add travel support form to the site.

Amended dates. Cut off 26th May, confirmed on 2nd June.

### CODE REVIEW

File: travel-support.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+layout: default
+title: Travel Support
+slug: travel-support
+---
+
+## Travel Support
+
+The organisers are arranging a travel support fund for those who would otherwise be unable to attend sustain in San Francisco on 19th June.  Travel support will cover the costs of ground and air transportation, accomodation and meals along with the cost of a ticket. 
+
+Sustain OSS is all about supporting open source maintainers. We want to hear from maintainers with a deep understanding of the issues and from those who have tested ideas to improve their own projects. With that in mind the organisers will prioritise support for those with experience of maintainting or sustianing and open source project.
```
Comment (RichardLitt) [inline travel-support.md]: maintianting > maintaining. 
sustianing > sustaining.

While we're at it - maybe American spelling of `organizers`?

**Comment by jdorfman** [comment]
@BenJam just saw this. Will review now

File: travel-support.md (hunk 1 above)
Comment (jdorfman) [inline travel-support.md]: agree on all 3.

**Comment by jdorfman** [comment]
@BenJam Looks good. Please `s/Sustain OSS/SustainOSS/g` & fix the spelling. Thanks!

**Pull request merged**